optaweb-vehicle-routing-backend/src/main/resources/application.properties

```diff
@@ -14,6 +14,7 @@
 # limitations under the License.
 #
 app.optimizer.timeout=1m
+app.region.country-codes=BE
 app.routing.osmfile=belgium-latest.osm.pbf
```

Comment:
"country-codes" is plural. So I would expect to be able to fill in multiple values, such as "BE, FR". Is that correct?
If it is, and I do fill in multiple values, can I fill in multiple osm files? I can't, as that parameter is not plural.

We have a convention that multiple values are comma separated.

Reply:
Also, "country-codes" and "osmfile" are not consistent in naming, these are:

- countryCodes and osmFile
- country-codes and osm-file
- countrycodes and osmfile

In any case, we should follow spring boot conventions there.

Reply:
Both styles (and couple additional ones) work thanks to Spring Boot's [relaxed binding](https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html#boot-features-external-config-relaxed-binding).

This time I went for `country-codes` because it was suggested by IntelliJ's auto completion. I'll consider that the preferred style and convert all properties mentioned in README and used in `application.properties` to that style.

Reply:
Yes, `app.region.country-codes` is plural and, for example, `app.region.country-codes=DE,AT,CH` is valid.

At this point it's not possible to load multiple OSM files at the same time. `app.routing.osm-file` property is singular and doesn't allow multiple, comma-separated values.

Given the above, it's  possible to load the whole Europe and restrict geosearch to Germany, Austria & Switzerland. I have briefly described this in the readme. I didn't want to go into too much detail as its implementation-dependent and might change soon with adding the possibility to externalize demo data sets.